docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -40,13 +40,14 @@ In order to configure the cluster according to these steps please follow this [O
 ### Kubevirt cluster wide configuration
 
 The pod running the VM (a.k.a. the virt-launcher pod) needs to run the DPDK optimized performance profile defined in the previous chapter. This is done by selecting the runtimeclassName in the pod running the VM.
-Since currently Kubevirt does not support setting the runtimeclassName per VM, the change needs to be on a cluster level, by using the [jsonpatch annotations](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#jsonpatch-annotations) on HCO:
+Since currently Kubevirt does not support setting the runtimeclassName per VM, the change needs to be on a cluster level, by patching the HCO CR:
 ```bash
-oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged \
-kubevirt.kubevirt.io/jsonpatch='[{"op": "add", "path": "/spec/configuration/defaultRuntimeClass", "value": <runtimeclass-name>}]'
+oc patch -n openshift-cnv hco kubevirt-hyperconverged \
+--type='json' -p='[{"op": "add", "path": "/spec/defaultRuntimeClass", "value":"<runtimeclass-name>"}]'
```

Comment:
Please consider adding a sentence like "You should change the `defaultRuntimeClass` field under the xxx object, one way to do it is:"

Reply:
DONE (I think)